test/integration/base.py

```diff
@@ -116,6 +125,11 @@ def use_profile(self):
 
         self._profile_config = profile_config
 
+    def set_packages(self):
```

Comment:
Copied straight outta [here](https://github.com/fishtown-analytics/dbt/blob/dev/wilt-chamberlain/test/integration/base.py#L310)
Which then made me think, is there a reason we are creating the DBTIntegrationTest class instead of importing it from dbt?

Reply:
I think I might have had to mess with some things (or thought I had to mess with some things). Importing the test class feels a bit like too tight of a coupling but that could be totally wrong.